src/reports/models/database/cassandra/cassandraConnector.js

```diff
@@ -88,3 +118,22 @@ function executeQuery(query, params, queryOptions) {
         return Promise.reject(new Error('Error occurred in communication with cassandra'));
     });
 }
+
+async function getReportsAndParse(query, params, queryOptions) {
```

Comment:
maybe rename to joinReportsWIthSubscribers

Reply:
done